doc/source/whatsnew/v1.4.0.rst

```diff
@@ -680,7 +680,7 @@ Indexing
 - Bug in indexing on columns with ``loc`` or ``iloc`` using a slice with a negative step with ``ExtensionDtype`` columns incorrectly raising (:issue:`44551`)
 - Bug in :meth:`IntervalIndex.get_indexer_non_unique` returning boolean mask instead of array of integers for a non unique and non monotonic index (:issue:`44084`)
 - Bug in :meth:`IntervalIndex.get_indexer_non_unique` not handling targets of ``dtype`` 'object' with NaNs correctly (:issue:`44482`)
--
+- Bug in :meth:`Series.loc.__setitem__` and :meth:`Series.loc.__getitem__` not raising when using multiple keys (:issue:`13831`)
```

Comment:
I assume you mean more keys than the Index has levels (in case of MultiIndex)?

Reply:
No, the problem that this issue solves is when using multiple keys for a series _without_ a multiindex